frontend/gulp/tasks/fileModules.json

```diff
@@ -1,4 +1,5 @@
 {
   "dashboard": "dashboard" , 
-  "agreement": "dashboard"
+  "agreement": "dashboard",
+  "authorise": "oauth"
```

Comment:
we either need to change the template name, or this and the link to the js file in the template html file.